### ISSUE TITLE
Crash when loading in carla

### ISSUE THREAD

**Issue by diovudau** [jpcima/ADLplug#32]
Version is LV2  adlplug-git-v1.0.0.beta.2.r12.6a23226-1 (Arch AUR Build from just now)

Loading in carla-jack-multi the first time works, it is playable.  Save, quit carla, load again in carla-jack-multi and it crashes on init.

Attached is the carla project file

[adlplugcrash.tar.gz](https://github.com/jpcima/ADLplug/files/2429420/adlplugcrash.tar.gz)

And here is my coredump:

```
           PID: 17368 (carla-jack-mult)
           UID: 1000 (nils)
           GID: 100 (users)
        Signal: 11 (SEGV)
     Timestamp: Fri 2018-09-28 21:09:35 CEST (1min 33s ago)
  Command Line: /usr/bin/python3 /usr/share/carla/carla-jack-multi --with-appname=/usr/bin/carla-jack-multi --with-libprefix=/usr adlplug.carxp
    Executable: /usr/bin/python3.7
 Control Group: /user.slice/user-1000.slice/session-1.scope
          Unit: session-1.scope
         Slice: user-1000.slice
       Session: 1
     Owner UID: 1000 (nils)
       Boot ID: 9c377b72deb24c2685904b5531de586b
    Machine ID: 2c70fcfc28cd474b885f71472e6fbb54
      Hostname: fyps
       Storage: /var/lib/systemd/coredump/core.carla-jack-mult.1000.9c377b72deb24c2685904b5531de586b.17368.1538161775000000.lz4
       Message: Process 17368 (carla-jack-mult) of user 1000 dumped core.
                
                Stack trace of thread 17368:
                #0  0x00007fa9e56e7155 _ZN12Bank_Manager11clear_banksEb (ADLplug.so)
                #1  0x00007fa9e56e691a _ZN21AdlplugAudioProcessor19setStateInformationEPKvi (ADLplug.so)
                #2  0x00007fa9e56dd30c _ZN14JuceLv2Wrapper15lv2RestoreStateEPFPKvPvjPmPjS4_ES2_j (ADLplug.so)
                #3  0x00007faa4d4de0cf n/a (libcarla_standalone2.so)
                #4  0x00007faa4d4a1036 _ZN12CarlaBackend11CarlaPlugin13loadStateSaveERKNS_14CarlaStateSaveE (libcarla_standalone2.so)
                #5  0x00007faa4d461d71 _ZN12CarlaBackend11CarlaEngine19loadProjectInternalERN5water11XmlDocumentE (libcarla_standalone2.so)
                #6  0x00007faa4d4641d0 _ZN12CarlaBackend11CarlaEngine11loadProjectEPKc (libcarla_standalone2.so)
                #7  0x00007faa518fe1c8 ffi_call_unix64 (libffi.so.6)
                #8  0x00007faa518fdc2a ffi_call (libffi.so.6)
                #9  0x00007faa57a4a6b5 _ctypes_callproc (_ctypes.cpython-37m-x86_64-linux-gnu.so)
                #10 0x00007faa57a4b040 n/a (_ctypes.cpython-37m-x86_64-linux-gnu.so)
                #11 0x00007faa5e9d185c _PyObject_FastCallKeywords (libpython3.7m.so.1.0)
                #12 0x00007faa5ea1dc97 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0)
                #13 0x00007faa5e9a52fb _PyFunction_FastCallKeywords (libpython3.7m.so.1.0)
                #14 0x00007faa5ea1d291 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0)
                #15 0x00007faa5e9a52fb _PyFunction_FastCallKeywords (libpython3.7m.so.1.0)
                #16 0x00007faa5ea1d291 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0)
                #17 0x00007faa5e95ef3b _PyFunction_FastCallDict (libpython3.7m.so.1.0)
                #18 0x00007faa5e96ddd8 _PyObject_Call_Prepend (libpython3.7m.so.1.0)
                #19 0x00007faa5e95f66b PyObject_Call (libpython3.7m.so.1.0)
                #20 0x00007faa5da6dda1 n/a (QtCore.so)
                #21 0x00007faa5da6e291 n/a (QtCore.so)
                #22 0x00007faa5da6e42d n/a (QtCore.so)
                #23 0x00007faa5da70e15 n/a (QtCore.so)
                #24 0x00007faa5da70f23 n/a (QtCore.so)
                #25 0x00007faa5d60bb4d _ZN11QMetaObject8activateEP7QObjectiiPPv (libQt5Core.so.5)
                #26 0x00007faa5d617ccc n/a (libQt5Core.so.5)
                #27 0x00007faa5d60c30b _ZN7QObject5eventEP6QEvent (libQt5Core.so.5)
                #28 0x00007faa57f2ee14 _ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent (libQt5Widgets.so.5)
                #29 0x00007faa57f366e1 _ZN12QApplication6notifyEP7QObjectP6QEvent (libQt5Widgets.so.5)
                #30 0x00007faa585d918f n/a (QtWidgets.so)
                #31 0x00007faa5d5e1c39 _ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent (libQt5Core.so.5)
                #32 0x00007faa5d63439a _ZN14QTimerInfoList14activateTimersEv (libQt5Core.so.5)
                #33 0x00007faa5d634c22 n/a (libQt5Core.so.5)
                #34 0x00007faa5cc033cf g_main_context_dispatch (libglib-2.0.so.0)
                #35 0x00007faa5cc04f89 n/a (libglib-2.0.so.0)
                #36 0x00007faa5cc04fce g_main_context_iteration (libglib-2.0.so.0)
                #37 0x00007faa5d634fc9 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5)
                #38 0x00007faa50c5fe12 n/a (libQt5XcbQpa.so.5)
                #39 0x00007faa5d5e08cc _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5)
                #40 0x00007faa5d5e8bc6 _ZN16QCoreApplication4execEv (libQt5Core.so.5)
                #41 0x00007faa585d7c1d n/a (QtWidgets.so)
                #42 0x00007faa5e9a5a88 _PyMethodDef_RawFastCallKeywords (libpython3.7m.so.1.0)
                #43 0x00007faa5e9a5d21 _PyCFunction_FastCallKeywords (libpython3.7m.so.1.0)
                #44 0x00007faa5ea1d89c _PyEval_EvalFrameDefault (libpython3.7m.so.1.0)
                #45 0x00007faa5e9a52fb _PyFunction_FastCallKeywords (libpython3.7m.so.1.0)
                #46 0x00007faa5ea18b92 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0)
                #47 0x00007faa5e95dee9 _PyEval_EvalCodeWithName (libpython3.7m.so.1.0)
                #48 0x00007faa5e95ede4 PyEval_EvalCodeEx (libpython3.7m.so.1.0)
                #49 0x00007faa5e95ee0c PyEval_EvalCode (libpython3.7m.so.1.0)
                #50 0x00007faa5ea8a8b4 n/a (libpython3.7m.so.1.0)
                #51 0x00007faa5ea8c4be PyRun_FileExFlags (libpython3.7m.so.1.0)
                #52 0x00007faa5ea8d745 PyRun_SimpleFileExFlags (libpython3.7m.so.1.0)
                #53 0x00007faa5ea8f337 n/a (libpython3.7m.so.1.0)
                #54 0x00007faa5ea8f810 _Py_UnixMain (libpython3.7m.so.1.0)
                #55 0x00007faa5ebf4223 __libc_start_main (libc.so.6)
                #56 0x0000557a8476d05e _start (python3.7)
                
                Stack trace of thread 17382:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17392:
                #0  0x00007faa5ec98638 __nanosleep (libc.so.6)
                #1  0x00007faa5ecc3c28 usleep (libc.so.6)
                #2  0x00007faa4d47c0da n/a (libcarla_standalone2.so)
                #3  0x00007faa4d44b146 n/a (libcarla_standalone2.so)
                #4  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #5  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17385:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17372:
                #0  0x00007faa5ecc0bb1 __poll (libc.so.6)
                #1  0x00007faa5902c180 n/a (libxcb.so.1)
                #2  0x00007faa5902de4b xcb_wait_for_event (libxcb.so.1)
                #3  0x00007faa50bc9c5a n/a (libQt5XcbQpa.so.5)
                #4  0x00007faa5d433f65 n/a (libQt5Core.so.5)
                #5  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #6  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17380:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17386:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17393:
                #0  0x00007faa5ecc640d syscall (libc.so.6)
                #1  0x00007faa42393b85 n/a (libjack.so.0)
                #2  0x00007faa42374b43 n/a (libjack.so.0)
                #3  0x00007faa42379b66 n/a (libjack.so.0)
                #4  0x00007faa42372dfc n/a (libjack.so.0)
                #5  0x00007faa42372cc0 n/a (libjack.so.0)
                #6  0x00007faa42372c8a n/a (libjack.so.0)
                #7  0x00007faa423704db n/a (libjack.so.0)
                #8  0x00007faa42391072 n/a (libjack.so.0)
                #9  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #10 0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17376:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17379:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17373:
                #0  0x00007faa5ecc0bb1 __poll (libc.so.6)
                #1  0x00007faa5cc04ee0 n/a (libglib-2.0.so.0)
                #2  0x00007faa5cc04fce g_main_context_iteration (libglib-2.0.so.0)
                #3  0x00007faa5d634fc9 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5)
                #4  0x00007faa5d5e08cc _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5)
                #5  0x00007faa5d429eb9 _ZN7QThread4execEv (libQt5Core.so.5)
                #6  0x00007faa56fb4ba6 n/a (libQt5DBus.so.5)
                #7  0x00007faa5d433f65 n/a (libQt5Core.so.5)
                #8  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #9  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17374:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa5d43471c _ZN14QWaitCondition4waitEP6QMutexm (libQt5Core.so.5)
                #2  0x00007faa5814d2f6 n/a (libQt5Widgets.so.5)
                #3  0x00007faa5d433f65 n/a (libQt5Core.so.5)
                #4  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #5  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17397:
                #0  0x00007faa5ecc0bb1 __poll (libc.so.6)
                #1  0x00007fa9e58eee7e _ZN4juce20InternalMessageQueue15sleepUntilEventEi (ADLplug.so)
                #2  0x00007fa9e56df547 _ZN19SharedMessageThread3runEv (ADLplug.so)
                #3  0x00007fa9e59226d0 _ZN4juce6Thread16threadEntryPointEv (ADLplug.so)
                #4  0x00007fa9e5922809 _ZN4juce21juce_threadEntryPointEPv (ADLplug.so)
                #5  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #6  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17396:
                #0  0x00007faa5eda9184 read (libpthread.so.0)
                #1  0x00007faa42392de0 n/a (libjack.so.0)
                #2  0x00007faa42396fd9 n/a (libjack.so.0)
                #3  0x00007faa42396d02 n/a (libjack.so.0)
                #4  0x00007faa42391072 n/a (libjack.so.0)
                #5  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #6  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17377:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17383:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17378:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17394:
                #0  0x00007faa5ec98638 __nanosleep (libc.so.6)
                #1  0x00007faa5ecc3c28 usleep (libc.so.6)
                #2  0x00007faa4d4876a2 n/a (libcarla_standalone2.so)
                #3  0x00007faa4d44b146 n/a (libcarla_standalone2.so)
                #4  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #5  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17384:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17375:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17391:
                #0  0x00007faa5eda9184 read (libpthread.so.0)
                #1  0x00007faa42392de0 n/a (libjack.so.0)
                #2  0x00007faa42396fd9 n/a (libjack.so.0)
                #3  0x00007faa42396d02 n/a (libjack.so.0)
                #4  0x00007faa42391072 n/a (libjack.so.0)
                #5  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #6  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17381:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa3a53be7c n/a (radeonsi_dri.so)
                #2  0x00007faa3a53bbb8 n/a (radeonsi_dri.so)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17389:
                #0  0x00007faa5eccbd77 epoll_wait (libc.so.6)
                #1  0x00007faa4d5766dc n/a (libcarla_standalone2.so)
                #2  0x00007faa4d576b7f n/a (libcarla_standalone2.so)
                #3  0x00007faa4d578adc _ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJZN7ableton9platforms4asio7ContextINS4_5posix13ScanIpIfAddrsENS3_4util7NullLogEEC4INS3_4link10ControllerISt8functionIFvmEESF_IFvNSD_5TempoEEENS4_5linux5ClockILi1EEESB_E23UdpSendExceptionHandlerEEET_EUlRN4asio10io_contextESP_E_St17reference_wrapperISS_ESP_EEEEE6_M_runEv (libcarla_standalone2.so)
                #4  0x00007faa5e0dc063 execute_native_thread_routine (libstdc++.so.6)
                #5  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #6  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17390:
                #0  0x00007faa5eda5afc pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa42391d34 n/a (libjack.so.0)
                #2  0x00007faa4238af09 n/a (libjack.so.0)
                #3  0x00007faa42391072 n/a (libjack.so.0)
                #4  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #5  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17388:
                #0  0x00007faa5eda5e5b pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007faa4d5707ab _ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJZN7ableton9platforms26LockFreeCallbackDispatcherISt8functionIFvvEENSt6chrono8durationIlSt5ratioILl1ELl1000EEEEEC4ES8_SD_EUlvE_EEEEE6_M_runEv (libcarla_standalone2.so)
                #2  0x00007faa5e0dc063 execute_native_thread_routine (libstdc++.so.6)
                #3  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #4  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17387:
                #0  0x00007faa5eccbd77 epoll_wait (libc.so.6)
                #1  0x00007faa4d5766dc n/a (libcarla_standalone2.so)
                #2  0x00007faa4d576b7f n/a (libcarla_standalone2.so)
                #3  0x00007faa4d576f60 _ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJZN7ableton9platforms4asio7ContextINS4_5posix13ScanIpIfAddrsENS3_4util7NullLogEEC4INSB_14DefaultHandlerEEET_EUlRN4asio10io_contextESD_E_St17reference_wrapperISG_ESD_EEEEE6_M_runEv (libcarla_standalone2.so)
                #4  0x00007faa5e0dc063 execute_native_thread_routine (libstdc++.so.6)
                #5  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #6  0x00007faa5eccba43 __clone (libc.so.6)
                
                Stack trace of thread 17395:
                #0  0x00007faa5ec98638 __nanosleep (libc.so.6)
                #1  0x00007faa5ecc3c28 usleep (libc.so.6)
                #2  0x00007faa4d44c81a n/a (libcarla_standalone2.so)
                #3  0x00007faa4d44b146 n/a (libcarla_standalone2.so)
                #4  0x00007faa5ed9fa9d start_thread (libpthread.so.0)
                #5  0x00007faa5eccba43 __clone (libc.so.6)

GNU gdb (GDB) 8.2
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-pc-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from /usr/bin/python3.7...(no debugging symbols found)...done.

warning: core file may not match specified executable file.
[New LWP 17368]
[New LWP 17382]
[New LWP 17392]
[New LWP 17385]
[New LWP 17372]
[New LWP 17380]
[New LWP 17386]
[New LWP 17393]
[New LWP 17376]
[New LWP 17379]
[New LWP 17373]
[New LWP 17374]
[New LWP 17397]
[New LWP 17396]
[New LWP 17377]
[New LWP 17383]
[New LWP 17378]
[New LWP 17394]
[New LWP 17384]
[New LWP 17375]
[New LWP 17391]
[New LWP 17381]
[New LWP 17389]
[New LWP 17390]
[New LWP 17388]
[New LWP 17387]
[New LWP 17395]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/usr/lib/libthread_db.so.1".
Core was generated by `/usr/bin/python3 /usr/share/carla/carla-jack-multi --with-appname=/usr/bin/carl'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007fa9e56e7155 in Bank_Manager::clear_banks (this=this@entry=0x0, notify=notify@entry=false)
    at /usr/src/debug/ADLplug/sources/bank_manager.cc:30
30	/usr/src/debug/ADLplug/sources/bank_manager.cc: No such file or directory.
[Current thread is 1 (Thread 0x7faa5ee07600 (LWP 17368))]



```

**Comment by jpcima** [comment]
Thanks for posting a detailed issue.
I attempted a fix in 85beaa9 but haven't tested, please try. From the trace it seems clear enough that state loading is attempted at startup, in a state where the processor is not fully ready.
The attempt is to force postponing the load after the creation of the chip player instance.

**Comment by diovudau** [comment]
I can confirm this fixed the problem.
Subsequently it fixed a crash when renaming the carla-rack with adlplug in it, which might rely on closing and reopening.
